app/src/organisms/ErrorRecoveryFlows/constants.ts

```diff
@@ -184,6 +186,16 @@ export const RECOVERY_MAP = {
       SKIP: 'skip',
     },
   },
+  LOAD_LABWARE_SHUTTLE_AND_RETRY: {
+    ROUTE: 'load-shuttle-and-retry',
+    STEPS: {
+      PREPARE_TRACK_FOR_HOMING: 'prepare-track-for-homing',
+      CLOSE_DOOR_AND_HOME: 'close-door-and-home',
+      MANUAL_REPLACE: 'load-shuttle',
```

Comment:
```suggestion
      MANUAL_REPLACE: 'manual-load-shuttle',
```
Keeps the semantic pattern.